lib/natural_person.ex

```diff
@@ -343,6 +343,18 @@ defmodule RfcFacil.NaturalPerson do
     end
   end
 
+  # Regresa la siguiente vocal, o si no hay, la segunda letra
+  defp paternal_surname_letter_2(paternal_surname) do
+    vowel = next_vowel(paternal_surname)
+
+    if vowel do
+      vowel
+    else
+      {next_letter, _} = String.next_grapheme(paternal_surname)
```

Comment:
Won't this crash if `paternal_surname` is empty at this point?

Reply:
Yes, this will (still) break for single-letter paternal surnames.